### PR TITLE
Plots path in color; blue = start, red = end

### PR DESCRIPTION
**Pull request by forea**
Tweaked so that user can see progression of trial, from start (blue) to end (red)

### CODE REVIEW

**Comment by kimtonyhyun** [comment]
Add a picture for others to see?

**Comment by forea** [comment]
Pretty colors!
screenshot from 2015-03-05 13 33 04

**Comment by kimtonyhyun** [comment]
Also, I thought that if you put the trial number in each subplot not as `title` but `ylabel`, then it would be easier to distinguish which image belongs to which trial.

**Comment by forea** [comment]
tried it out, hard to read

**Comment by kimtonyhyun** [comment]
Ok, looks good.

File: behavior/MouseTracking/FindTrials/mouse_path_plotter.m [hunk 1]
```diff
@@ -47,9 +48,12 @@
         set(gca,'XTick',[],'YTick',[]);
         colormap gray;
         hold on
-        plot(centroids(trial_start:trial_end,1),...
-             centroids(trial_start:trial_end,2),...
-             'g-');
+        
+        % plot centroids in color
+        for i = trial_start:trial_end
+            color_options = jet(size(centroids(trial_start:trial_end,1),1));
```
Comment (kimtonyhyun) [inline behavior/MouseTracking/FindTrials/mouse_path_plotter.m]: Actually, this line should go outside of the `for i = 1:trial_start:trial_end` loop.

**Pull request merged**